db/mysql/client.go

```diff
@@ -2,22 +2,24 @@ package mysql
 
 import (
 	"fmt"
+	"github.com/spf13/viper"
 	"strings"
 
 	"github.com/go-sql-driver/mysql"
 	"github.com/jmoiron/sqlx"
 	"github.com/pkg/errors"
 
+	"github.com/ahab94/ApnaSchool/configuration"
 	"github.com/ahab94/ApnaSchool/db"
 	"github.com/ahab94/ApnaSchool/models"
 )
 
 const (
-	databaseName     = "apna_school"
+	databaseName     = viper.GetString(configuration.DbName)
```

Comment:
We cannot instantiated consts like this. we are not going to use consts now that we have these values configurable. use viper directly where needed.